### PR TITLE
docs(toolbar): adding information regarding column hiding and pinning

### PR DESCRIPTION
**Pull request by rmkrmk**
- adding information regarding column hiding and column pinning
- adding information regarding the "dropDownsMaxHeight" property
- fixing a typo

### CODE REVIEW

File: en/components/grid_toolbar.md [hunk 1]
```diff
@@ -27,11 +27,13 @@ The IgxGrid's toolbar is shown using the grid's `showToolbar` property - just se
...
 
-The IgxGrid's toolbar can be configured to allow IgxGrid's data exporting in different formats - MS Excel and CSV. You can enable each exporting format independently by setting its dedicated boolean property to true:
+The IgxGrid's toolbar can be configured to allow columns hiding, columns pinning and exporting data to MS Excel and CSV. You can enable each feature independently by setting its dedicated boolean property to true:
+- for column hiding set `columnHiding` to `true`
+- for column pinning set `columnPinning` to `true`
 - for MS Excel export set the grid's `exportExcel` property to `true`
-- for CSV export set the grid's `exportCsv` property to `true`
+- for CSV export set the grid's `exportCsv` property to `true`.
 
 The available export options are hosted in a drop down which is shown when the main export button on the toolbar is pressed. There are properties for configuring each button's text and they are listed in the API section below.
```
Comment (gedinakova) [inline en/components/grid_toolbar.md]: We are no longer describing the export options only, so we should either remove or update this paragraph.

File: en/components/grid_toolbar.md [hunk 2]
```diff
@@ -43,12 +45,14 @@ The available export options are hosted in a drop down which is shown when the m
 <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="grid-toolbar-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
-The following code snippet demonstrates how to enable the MS Excel export and CSV export features on the toolbar and also set titles for the exporting buttons:
+The following code snippet demonstrates how to enable all features of the of the toolbar:
```
Comment (gedinakova) [inline en/components/grid_toolbar.md]: 'of the' twice ->enable all features **of the of the** toolbar:

File: en/components/grid_toolbar.md [hunk 3]
```diff
@@ -77,7 +81,7 @@ export class AppModule {}
 When the user initiates an export process the IgxGrid's `onToolbarExporting` event is emitted. This event exposes both the export options and the exporter itself. This allows you to configure the export process.
```
Comment (gedinakova) [inline en/components/grid_toolbar.md]: We should add some intro, for example, 'In addition to simply switching on and off the toolbar features, the export process can be further configured in the `onToolbarExporting` event which is emitted when the user initiates the export process.'

File: en/components/grid_toolbar.md [hunk 4]
```diff
@@ -135,8 +139,15 @@ The Grid Toolbar service has a few more APIs to explore, which are listed below.
 The following inputs are available on the **IgxGrid** component:
 | Name | Type | Description |
 | :--- | :--- | :--- |
+| `toolbar` | IgxGridToolbarComponent | A reference to the toolbar component. |
 | `showToolbar` | boolean | Show or hide the grid's toolbar. |
 | `toolbarTitle` | string | Sets the toolbar title. |
+| `columnHiding` | boolean | Show or hide the column hiding feature. |
```
Comment (gedinakova) [inline en/components/grid_toolbar.md]: I suggest we order these as follows:
- columnHiding
- columnHidingTitle
- hiddenColumnsText 
Same for Pinning

**Comment by yoshimis** [comment]
Code part should be updated in JP file.

File: en/components/grid_toolbar.md (hunk 4 above)
Comment (rmkrmk) [reply en/components/grid_toolbar.md]: Done

File: en/components/grid_toolbar.md (hunk 3 above)
Comment (rmkrmk) [reply en/components/grid_toolbar.md]: Done

File: en/components/grid_toolbar.md (hunk 2 above)
Comment (rmkrmk) [reply en/components/grid_toolbar.md]: Done

File: en/components/grid_toolbar.md (hunk 1 above)
Comment (rmkrmk) [reply en/components/grid_toolbar.md]: Done

File: en/components/grid_toolbar.md [hunk 5]
```diff
@@ -74,10 +78,10 @@ export class AppModule {}
 
 #### Customizing the Export
 
-When the user initiates an export process the IgxGrid's `onToolbarExporting` event is emitted. This event exposes both the export options and the exporter itself. This allows you to configure the export process.
+In addition to simply switching on and off the toolbar features, the export process can be further configured in the `onToolbarExporting` event which is emitted when the user initiates the export process. This event exposes both the exporter and the exporter's options. This allows you to configure the export process.
```
Comment (gedinakova) [inline en/components/grid_toolbar.md]: 'This allows you to configure the export process.' sentence should be removed now.

**Pull request merged**